src/index/update.rs

```diff
@@ -223,6 +223,12 @@ impl IndexUpdater {
 				directory_artist = tags.artist.as_ref().cloned();
 			}
 
+			let artwork_path = if tags.artwork {
```

Comment:
I see an issue where a collection using only embedded album art would end up with no artwork associated with directories at all. That is because the `// Insert directory` block below ignores the artwork found in tags.

I would suggest using the first embedded artwork found in a song for the directory artwork.